<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Joystick Configuration</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="joystickScan">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Scan</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Joystick Name:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="joystickName">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QComboBox" name="joystickSelect">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0">
           <widget class="QSplitter" name="splitter">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="QWidget" name="gridLayoutWidget">
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="0" column="1">
                 <widget class="QSlider" name="rollAxisValue">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="minimum">
                   <number>-180</number>
                  </property>
                  <property name="maximum">
                   <number>180</number>
                  </property>
                  <property name="tracking">
                   <bool>true</bool>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_5">
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Pitch</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QSlider" name="pitchAxisValue">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="minimum">
                   <number>-180</number>
                  </property>
                  <property name="maximum">
                   <number>180</number>
                  </property>
                  <property name="tracking">
                   <bool>true</bool>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QPushButton" name="detectRoll">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Detect</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="QPushButton" name="detectPitch">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Detect</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Plain</enum>
                  </property>
                  <property name="text">
                   <string>Roll</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="0" column="1">
               <layout class="QGridLayout" name="gridLayout_4">
                <item row="1" column="0">
                 <widget class="QLabel" name="label_6">
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Thrust</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QSlider" name="thrustAxisValue">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="minimum">
                   <number>0</number>
                  </property>
                  <property name="maximum">
                   <number>2000</number>
                  </property>
                  <property name="tracking">
                   <bool>true</bool>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_7">
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Yaw</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QSlider" name="yawAxisValue">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="minimum">
                   <number>-180</number>
                  </property>
                  <property name="maximum">
                   <number>180</number>
                  </property>
                  <property name="tracking">
                   <bool>true</bool>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QPushButton" name="detectYaw">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Detect</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="QPushButton" name="detectThrust">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Detect</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="1" column="0" colspan="2">
               <widget class="Line" name="line_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="gridLayoutWidget_2">
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="1" column="2">
               <widget class="QCheckBox" name="pitchPos">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Pitch +</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QPushButton" name="detectPitchPos">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Detect</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QCheckBox" name="rollNeg">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Roll -</string>
                </property>
               </widget>
              </item>
              <item row="2" column="4">
               <widget class="QPushButton" name="detectRollPos">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Detect</string>
                </property>
               </widget>
              </item>
              <item row="3" column="2">
               <widget class="QCheckBox" name="pitchNeg">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Pitch -</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QPushButton" name="detectRollNeg">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Detect</string>
                </property>
               </widget>
              </item>
              <item row="2" column="3">
               <widget class="QCheckBox" name="rollPos">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Roll +</string>
                </property>
               </widget>
              </item>
              <item row="4" column="2">
               <widget class="QPushButton" name="detectPitchNeg">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Detect</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="0" column="1">
             <widget class="QPushButton" name="detectAltHold">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Detect</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QCheckBox" name="exitapp">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Exit App</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="althold">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Altitude Hold</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="killswitch">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Killswitch</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QCheckBox" name="resevered1">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Resevered</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QCheckBox" name="resevered2">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Resevered</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QCheckBox" name="resevered3">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Resevered</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="detectKillswitch">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Detect</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="detectExitapp">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Detect</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="detectResevered1">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Detect</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QPushButton" name="detectResevered2">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Detect</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QPushButton" name="detectResevered3">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Detect</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Profile name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="configFileSelect">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="configLoad">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Load And Config</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="configSave">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Serial Port Configuration</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="7" column="0">
        <widget class="QPushButton" name="serialOpen">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Open</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="serialPortScan">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Scan</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QPushButton" name="serialBrowserClear">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Data bits</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Stop bits</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Flow control</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QPushButton" name="serialClose">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>BaudRate</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Parity</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QComboBox" name="serialPortSelect">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QComboBox" name="baudRateSet">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>9600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>38400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>115200</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QComboBox" name="dataBitsSet">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QComboBox" name="paritySet">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Even</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Odd</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mark</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Space</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QComboBox" name="stopBitsSet">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1.5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QComboBox" name="flowControlSet">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RTS/CTS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>XON/XOFF</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="3" rowspan="8">
        <widget class="QTextBrowser" name="serialTextBrowser"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
